config/application.rb

```diff
@@ -10,6 +10,7 @@
 require "action_view/railtie"
 require "sprockets/railtie"
 # require "rails/test_unit/railtie"
+require 'dotenv' ; Dotenv.load ".env.local", ".env.#{Rails.env}"
```

Comment:
Do not use semicolons to terminate expressions.<br>Space found before semicolon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.